src/Nox.Generator/Application/NoxWebApplicationExtensionGenerator.cs

```diff
@@ -13,69 +16,84 @@ public void Generate(SourceProductionContext context, NoxSolutionCodeGeneratorSt
     {
         context.CancellationToken.ThrowIfCancellationRequested();
 
-        var code = new CodeBuilder($"Application.NoxWebApplicationExtensions.g.cs", context);
+        var scribanTemplate = Template.Parse(
+            @"// Generated
```

Comment:
Also I wasn't sure if I should do this `Template.Parse` here or use the `TemplateCodeBuilder` somehow.
Is it correct to parse and render the template here?